### PR TITLE
Code of Conduct

### LINKED ISSUE #10: [Feature] Create code of conduct
## Feature description

Add a CODE_OF_CONDUCT.md file to the project

### PR DESCRIPTION
**Pull request by JuanitaCathy**

### CODE REVIEW

File: src/index.css [hunk 1]
```diff
@@ -26,5 +26,5 @@ code {
 	align-items: center;
 	background-size: 1%;
 	background-image: url("data:image/svg+xml,%3Csvg width='20' height='20' viewBox='0 0 20 20' xmlns='http://www.w3.org/2000/svg'%3E%3Cg fill='%23000000' fill-opacity='0.2' fill-rule='evenodd'%3E%3Ccircle cx='3' cy='3' r='3'/%3E%3Ccircle cx='13' cy='13' r='3'/%3E%3C/g%3E%3C/svg%3E");
-    cursor:grab;
+
```
Comment (eduardconstantin) [inline src/index.css]: I think you are on an old branch, please sync the fork
image

**Review by eduardconstantin — CHANGES_REQUESTED** [review]

**Comment by eduardconstantin** [comment]
And don't delete the syle.css file

File: src/index.css (hunk 1 above)
Comment (JuanitaCathy) [reply src/index.css]: It shows i have to delete 3 commits, do i delete it?

Comment (eduardconstantin) [reply src/index.css]: I guess, I think are those you made in this PR

**Comment by JuanitaCathy** [comment]
So i create a new branch now and add the Code of Conduct again? I just realised i didnt create a new branch before this PR, sorry about that

**Comment by JuanitaCathy** [comment]
And im getting merge conflicts

**Pull request closed without merging**